daemon/hostdaemon.go

```diff
@@ -26,11 +29,11 @@ type HostDaemon struct {
 	cniserver *cniserver.Server
 }
 
-func (d *HostDaemon) CreateBridgePort(pf int, vf int, vlan int) (error) {
+func (d *HostDaemon) CreateBridgePort(pf int, vf int, vlan int, mac string) (error) {
```

Comment:
(Comment) Remember how we might want to have a json for the name... We are fast approaching needing this.